client/client.go

```diff
@@ -183,9 +183,9 @@ func (cli *Client) Close() error {
 
 // getAPIPath returns the versioned request path to call the api.
 // It appends the query parameters to the path if they are not empty.
-func (cli *Client) getAPIPath(p string, query url.Values) string {
+func (cli *Client) getAPIPath(p string, query url.Values, ignoreVersion bool) string {
```

Comment:
I wonder if it better to have a `ignoreVersion` bool or handle the `_ping` endpoint by itself in the method :stuck_out_tongue: